packages/core/src/components/ListItemAvatar/ListItemAvatar.tsx

```diff
@@ -27,6 +36,7 @@ const ListItemAvatar: VibeComponent<ListItemAvatarProps> & { components?: typeof
           type={Avatar.types.IMG}
           size={Avatar.sizes.SMALL}
           className={cx(styles.avatar, avatarClassName)}
+          {...avatarProps}
```

Comment:
should I omit `className` and `id` as they're applied to the `Component`?

Reply:
They are already omitted while spreading the props so no need

Reply:
Oh, you're absolutely right, my bad 🤦